internal/metrics/metricsManager.go

```diff
@@ -305,6 +311,7 @@ func NewMetricsManager(registry prometheus.Registerer) *MetricsManager {
 }
 
 func (m *MetricsManager) ClusterRegistered(clusterVersion string, clusterID strfmt.UUID, emailDomain string) {
+	//MGMT-4526 TODO: remove this scrap after ELK dashboards are verified
```

Comment:
Where is the transfer to ELK? Why don't you generate an event instead?

Reply:
We decided that we allow few weeks for the dashboard to run on ELK, monitor them and debug them, and only then cancel the reports to Prometheus. I ma marking the places for deletion now with TODO label so I won't need to do the research again

Reply:
Yes this I understand but you "future deleted metrics" but didn't add any events instead.
Also as I mentioned https://github.com/openshift/assisted-service/pull/1560#issuecomment-831256621 I think we should have some tests for it and not just checking manually in elastic.

Reply:
These are special cases where we already have measurement in ELK on those metrics based on existing events. So, In these cases we just need to remove the special metric reports to Prometheus without adding a thing

Reply:
Ack.